terror/terror.go

```diff
@@ -39,10 +35,8 @@ type ErrCode int
 
 // Executor error codes.
 const (
-	CodeUnknown                  ErrCode = -1
-	CodeCommitNotInTransaction           = 1
-	CodeRollbackNotInTransaction         = 2
-	CodeExecResultIsEmpty                = 3
+	CodeUnknown           ErrCode = -1
+	CodeExecResultIsEmpty         = 3
```

Comment:
`CodeCommitNotInTransaction` is useless? And Is other errors use CodeExecResultIsEmpty?  If not, set it to 1